pkg/api/validation/validation.go

```diff
@@ -2844,10 +2799,16 @@ func ValidateNodeUpdate(node, oldNode *api.Node) field.ErrorList {
 	// Clear status
 	oldNode.Status = node.Status
 
+	// update taints
```

Comment:
currently, admission is used on pod creation to verify the node selector and toleration uses are authorized. taints/tolerations are one tool to keep user pods off of infrastructure nodes. allowing mutation means revisiting admission plugins and reauthorizing pod specs on update.